src/SqlProcessor.php

```diff
@@ -112,6 +115,11 @@ function ($matches) use ($args, &$j, $last) {
 	 */
 	public function processModifier($type, $value)
 	{
+		if (isset($this->handlers[$value])) {
+			$handler = $this->handlers[$type];
+			return call_user_func_array($handler, func_get_args());
+		}
+
```

Comment:
intentionally called before default modifiers, but it's up for discussion. Leaving it up here may allow users to introduce security issues, such as when incorrectly rewriting `%s` to `return $string`.